src/v0/destinations/active_campaign/transform.js

```diff
@@ -286,7 +282,7 @@ const customFieldProcessor = async (message, category, destination) => {
     if (Array.isArray(fieldInfo[key])) {
       fPayload = '||';
       fieldInfo[key].map((fv) => {
-        fPayload = `${fPayload}${fv}||`;
+        fPayload = `${fPayload}${fv}|| `;
```

Comment:
```suggestion
        fPayload = `${fPayload}${fv}||`;
```
Please check if this is valid

Reply:
Yes this is valid case. Checked